pkg/controller/metrics/provision_underway_collector.go

```diff
@@ -30,6 +30,11 @@ var (
 type provisioningUnderwayCollector struct {
 	client client.Client
 
+	// minDuration, when non-zero, is the minimum duration afer which clusters provisioning
+	// will start becomming part of thismetric. When set to zero, all clusters provisioning
```

Comment:
nit - but we tend to ignore doc errors later on
`will start becoming part of this metric`*